index.js

```diff
@@ -88,6 +88,10 @@ function setupWebSocketProxy (fastify, options) {
     oldClose.call(this, done)
   }
 
+  server.on('error', (err) => {
+    fastify.log.error(err)
+  })
```

Comment:
Should the user of this module be able to specify a handler for this case?

Reply:
I'm not sure if it'll be useful.
Proxy already handles connection errors:
https://github.com/fastify/fastify-http-proxy/blob/328fd6bd06be7eab4befe327defc2c049f0b6da3/index.js#L52

This particular handler is to prevent `unhandledRejection` on WS Server start and it must be always provided